NSrtm.Core/FileExtensions/Pgm/Loaders/PgmDataLoaderFromRaw.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace NSrtm.Core
+{
+    internal sealed class PgmDataLoaderFromRaw : PgmDataLoaderFromFileStreamBase
+    {
+
+        public PgmDataLoaderFromRaw([NotNull] IPgmPathResolver pathResolver) : base(pathResolver)
+        {
+        }
+
+        protected override byte[] LoadAdfDataFromFile(string filePath)
+        {
+            using (var fileStream = File.Open(filePath,FileMode.Open,FileAccess.Read,FileShare.Read))
```

Comment:
Please at least reformat code you are commiting (missing spaces are only indicator).